data.js

```diff
@@ -304,10 +304,10 @@ const dataset = [
     ],
   },
   {
-    title: "Html",
-    slug: "html",
-    addedBy: "Honeshwar-Thakur",
-    cardImagePath: "./RandomSize/html.png",
+    title: "MDN",
+    slug: "mdn",
+    addedBy: "dipayansarkar47",
+    cardImagePath: "./RandomSize/mdn.png",
```

Comment:
resolve this issue,
your changes, overriding others commits

Reply:
okay.

Reply:
Check now, github commit it as Merge branch 'main' into developer.